### PR TITLE
DDW-981 Fix incorrect design of disabled inputs

### PR DESCRIPTION
**Pull request by przemyslaw-wlodek**
This PR fixes disabled input design for `cardano` and `light-blue` themes.

## Screenshots

Before:
image

After:
image

## Testing Checklist

- [Slack QA thread](https://github.com/input-output-hk/daedalus/pull/2891)
- [ ] Test

---

### Basics

- [x] PR assigned to the PR author(s)
- [x] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [x] If there are UI changes, Alexander Rukin assigned as an additional reviewer
- [ ] All visual regression testing has been reviewed (assign `run Chromatic` label to PR to trigger the run)
- [x] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [x] PR link is added to a Jira ticket, ticket moved to In Review
- [x] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [x] PR has a good description that summarizes all changes
- [x] PR contains screenshots (in case of UI changes)
- [x] CHANGELOG entry has been added to the top of the appropriate section (_Features_, _Fixes_, _Chores_) and is linked to the correct PR on GitHub
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality

- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with typescript types
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing

- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review

- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -71,8 +71,7 @@
 
 ### Fixes
 
-- Fixed styling of the incentivized testnet rewards wallet dropdown ([PR 2816](https://github.com/input-output-hk/daedalus/pull/2816/files))
-- Fixed blockchain verification progress text ([PR 2840](https://github.com/input-output-hk/daedalus/pull/2840))
+- Fixed blockain verification progress text ([PR 2840](https://github.com/input-output-hk/daedalus/pull/2840))
```
Comment (szymonmaslowski) [inline CHANGELOG.md:74]: I am not sure about amending Changelog history. I know this was done automatically by git however I am not sure this is the right way..

File: package.json [hunk 2]
```diff
@@ -248,7 +248,7 @@
     "react-intl": "2.7.2",
     "react-lottie": "1.2.3",
     "react-markdown": "4.3.1",
-    "react-polymorph": "1.0.3",
+    "react-polymorph": "1.0.2",
```
Comment (307sagar) [inline package.json:252]: is there any specific reason for lowering the react-polymorph version? Is it particular to the design aspect of the UI-components like dropdowns in this case.

Comment (przemyslaw-wlodek) [reply package.json:252]: Please take a look at the comments in the ticket.

**Review by alexander-rukin — APPROVED** [review]

**Review by dmitrii-gaico — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -72,7 +72,6 @@
 
 ### Fixes
 
-- Fixed styling of the incentivized testnet rewards wallet dropdown ([PR 2816](https://github.com/input-output-hk/daedalus/pull/2816/files))
```
Comment (DominikGuzei) [inline CHANGELOG.md:75]: why remove an existing changelog entry instead of adding a new one for this PR?

File: package.json (hunk 2 above)
Comment (DominikGuzei) [reply package.json:252]: @przemyslaw-wlodek are we sure that this **only** reverts this specific change and not other stuff as well?

**Review by DominikGuzei — CHANGES_REQUESTED** [review]
I have added some questions

**Review by gabriela-ponce — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (renanvalentin) [reply package.json:252]: Yeah. That's my concern as well. Looks like it also contains changes from 1.0.2:
https://github.com/input-output-hk/react-polymorph/pull/199

**Review by renanvalentin — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (renanvalentin) [reply package.json:252]: I tested it locally and seems to be fine

**Review by renanvalentin — APPROVED** [review]

**Review by miorsufianiohk — APPROVED** [review]
LGTM. Tested on 21048. Great work @przemyslaw-wlodek 👍

File: CHANGELOG.md (hunk 3 above)
Comment (przemyslaw-wlodek) [reply CHANGELOG.md:75]: The change is being reverted and it was not released. It was basically a mistake that someone put it there in the changelog as it should be in vNext.

File: package.json (hunk 2 above)
Comment (przemyslaw-wlodek) [reply package.json:252]: Yes, The changes in changelog are intentional and discussed with @danielmain and @dmitrii-gaico.
There's nothing else than a revert aparat from that.

File: CHANGELOG.md (hunk 3 above)
Comment (danielmain) [reply CHANGELOG.md:75]: @DominikGuzei this was a mistake, now we are correcting it

**Pull request merged**